### DISCUSSION TITLE
I got banned from the discord server because of running a command?

### DISCUSSION THREAD

**Discussion by hdfsyu** [OhMyGuus/BetterCrewLink#288] [General]
title is self-explanatory, I ran a command in #bot-commands and the command I used was ;shop buy then I typed 2 and it banned me from the server!?

**Comment by MatadorProBr** [comment]
What is your account ID? So I can view your ban and if you got banned literally for this reason, I will unban you

**Reply by hdfsyu** [reply]
im the dis#0522

**Reply by MatadorProBr** [reply]
Ok, it was a mistake of our bot that he banned you for switching the ";shop" too fast that he detected as spam, you are now unbanned so you can join again using this invite: https://discord.gg/qDqTzvj4SH

**Reply by hdfsyu** [reply]
well uh I got banned again looks like (I tried rejoining nothin happened! Just said you cant get into the server) I just answered 1 (in the role shaq), probably because of the same reason....

**Reply by MatadorProBr** [reply]
Yeah it seems the bot is banning you because you are sending 15 messages in 1 minute, also I unbanned you (again) so you can join in the same invite and try to avoid sending more that 15 messages in 1 minute
